external/runtime/runtime.depproj

```diff
@@ -25,22 +25,22 @@
 
   <ItemGroup Condition="'$(TargetGroup)'!='uapaot'">
     <PackageReference Include="Microsoft.NETCore.Platforms">
-      <Version>$(PlatformPackageVersion)</Version>
+      <Version>$(MicrosoftNETCorePlatformsPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>$(CoreClrPackageVersion)</Version>
+      <Version>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.TestHost">
-      <Version>$(CoreClrPackageVersion)</Version>
+      <Version>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="runtime.native.System.Data.SqlClient.sni">
-      <Version>4.4.0</Version>
+      <Version>$(RuntimeNativeSystemDataSqlClientSniPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.DotNetHost">
-      <Version>2.0.1-servicing-25615-03</Version>
+      <Version>$(MicrosoftNETCoreDotNetHostPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>2.0.1-servicing-25615-03</Version>
+      <Version>$(MicrosoftNETCoreDotNetHostPackageVersion)</Version>
```

Comment:
The version property should be separate since it is a different package.  The `DotNetHost` and `DotNetHostPolicy` packages can and will version separately.

Reply:
I agree these should be separate versions. These ones are also kind of special in that they don't match any other versions because we end up pulling a higher version to get RH support. Hopefully that doesn't cause us problems but it might.

Reply:
Splitting them out.